redis/asyncio/sentinel.py

```diff
@@ -122,6 +147,7 @@ def __init__(self, service_name, sentinel_manager, **kwargs):
         self.sentinel_manager = sentinel_manager
         self.master_address = None
         self.slave_rr_counter = None
+        self._request_id_to_replica_address = {}
```

Comment:
Would it make sense to use a more general name, e.g. instead of `request_id` use `context_id`? Would it express better the fact that you are basically trying to run several commands in the same context?